libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -422,6 +433,10 @@ bool AP_Follow::get_target_location_and_velocity(Location &loc, Vector3f &vel_ne
     }
     vel_ned = _estimate_vel_ned_ms;
 
+    if (_alt_type == Location::AltFrame::ABOVE_HOME) {
```

Comment:
Are these changes ok @lthall ? Tridge wanted me to check with you.

Reply:
Modified to set _ait_type regardless as I want it also to work for ABOVE_TERRAIN.